web-console/developer/advanced/advanced.md

```diff
@@ -12,6 +12,49 @@ When clicking on a specific project, Projects remains selected in the navigation
 
 ![Add project details](img/image2.png)
 
+## Project Access
+The Project Access page allows users to add or remove access to a project.
+
+![project access page](img/membership1.png)
```

Comment:
not sure what you mean about "primary navigation".  All of our pages in the dev perspective ( except search & events ) have the secondary masthead.  So we always force users to be in context of a project - so it wouldn't be a tab on the project details page because you are already in context :)

Reply:
I got confused here too - I think the first (and last) two images of this PR just need to be updated with the new nav that's being proposed. It doesn't seem like a dynamically-changing nav is part of this design.

![project-access](https://user-images.githubusercontent.com/9122899/67065640-1f2f3f80-f13d-11e9-9877-96363da957d3.png)

Reply:
Yep, that was my confusion too. I think if that first screenshot is updated it makes total sense. The user is already looking at a specific project, so they can navigate right to project details and project access at that point.